### PR TITLE
UI helper blueprint library

### PR DESCRIPTION
**Pull request by ThibNach**
added function to get widget associated to a list view item
added function to get focused widget from an array of widgets

### CODE REVIEW

File: Source/CoreExtensions/Private/BlueprintLibraries/UCoreExtUIBlueprintLibrary.cpp [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿#include "BlueprintLibraries/UCoreExtUIBlueprintLibrary.h"
+
+#include "Kismet/GameplayStatics.h"
```
Comment (Frickerson) [inline Source/CoreExtensions/Private/BlueprintLibraries/UCoreExtUIBlueprintLibrary.cpp]: <>

File: Source/CoreExtensions/Public/BlueprintLibraries/UCoreExtUIBlueprintLibrary.h [hunk 2]
```diff
@@ -0,0 +1,21 @@
+﻿#pragma once
+
+#include "CoreMinimal.h"
```
Comment (Frickerson) [inline Source/CoreExtensions/Public/BlueprintLibraries/UCoreExtUIBlueprintLibrary.h]: <>

File: Source/CoreExtensions/Public/BlueprintLibraries/UCoreExtUIBlueprintLibrary.h [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+#include "Kismet/BlueprintFunctionLibrary.h"
```
Comment (Frickerson) [inline Source/CoreExtensions/Public/BlueprintLibraries/UCoreExtUIBlueprintLibrary.h]: <>

**Review by Frickerson — CHANGES_REQUESTED** [review]

File: Source/CoreExtensions/Private/BlueprintLibraries/UCoreExtUIBlueprintLibrary.cpp (hunk 1 above)
Comment (ThibNach) [reply Source/CoreExtensions/Private/BlueprintLibraries/UCoreExtUIBlueprintLibrary.cpp]: Done

File: Source/CoreExtensions/Public/BlueprintLibraries/UCoreExtUIBlueprintLibrary.h (hunk 2 above)
Comment (ThibNach) [reply Source/CoreExtensions/Public/BlueprintLibraries/UCoreExtUIBlueprintLibrary.h]: Done

File: Source/CoreExtensions/Public/BlueprintLibraries/UCoreExtUIBlueprintLibrary.h (hunk 3 above)
Comment (ThibNach) [reply Source/CoreExtensions/Public/BlueprintLibraries/UCoreExtUIBlueprintLibrary.h]: Done

**Review by Frickerson — APPROVED** [review]

**Review by MaxDaniels — APPROVED** [review]

**Review by Fredaly — APPROVED** [review]

**Pull request merged**